File: tests/expected/built_ins.go
package main

import (
	"fmt"
	"math"
)

func DefaultBuiltins() {
	var a string = ""
	var b bool = false
	var c int = 0
	var d float64 = 0.0
	if !(a == "") {
		panic("assert")
	}
	if !(b == false) {
		panic("assert")
	}
	if !(c == 0) {
		panic("assert")
	}
	if !(d == 0.0) {
		panic("assert")
	}
}

func main() {
	var a int = int(math.Max(1, 2))
	fmt.Printf("%v\n", a)
	var b int = int(math.Min(1, 2))
	fmt.Printf("%v\n", b)
	var c int = int(math.Min(1.0, 2.0))
	fmt.Printf("%v\n", c)
}

File: tests/expected/built_ins.nim
proc default_builtins() =
  let a = ""
  let b = false
  let c = 0
  let d = 0.0
  assert(a == "")
  assert(b == false)
  assert(c == 0)
  assert(d == 0.0)

proc main() =
  let a = max(1, 2)
  echo a
  let b = min(1, 2)
  echo b
  let c = int(min(1.0, 2.0))
  echo c

main()
